frontend/src/components/InnerNavbar/InnerNavbar.scss

```
@@ -0,0 +1,9 @@
+
+
+.SideBar {
+  width: 240px;
+  height: 100%;
+  padding: 20px;
+  border-top-left-radius: 20px;
+  background-color: rgba($color: #090d2c, $alpha: 0.5);
+  }
```

ensure there is one line at the end of the file to avoid EOF errors that can happen between operating systems. In addition, ensure that the `}` is unindented 
```suggestion
}

```